components/boards/espressif_saola_1_wrover/board.h

```diff
@@ -56,7 +56,7 @@
 
 // LED for indicator and writing flash
 // If not defined neopixel will be use for flash writing instead
-#define PIN_LED               33
+#define PIN_LED               36
```

Comment:
saola doesn't has regular led, this should be commented out. It is my bad somehow I got it committed to the code base. I think I intended to test with it previously, but ended up pulling to other work.